src/sidebar/util/test/thread-sorting-test.js

```diff
@@ -0,0 +1,125 @@
+import { sortFns } from '../thread-sorting';
+import { $imports } from '../thread-sorting';
+
+describe('sidebar/util/thread-sorting', () => {
+  let fakeRootAnnotations;
+  let fakeLocation;
+
+  beforeEach(() => {
+    // The thread argument passed to `Newest` or `Oldest` sorting functions
+    // gets wrapped with an additional Array by `*RootAnnotationDate` before
+    // being passed on to `rootAnnotations`. This unwraps that extra array
+    // and returns the original first argument to `*RootAnnotationDate`
+    fakeRootAnnotations = sinon.stub().callsFake((...args) => args[0][0]);
```

Comment:
```suggestion
    fakeRootAnnotations = sinon.stub().callsFake(threads => threads[0]);
```

To make things more obvious for a future reader at a glance.